pkg/controller/certificates/trigger/policies/policies.go

```diff
@@ -178,17 +179,34 @@ func currentSecretValidForSpec(input Input) (string, string, bool) {
 	return "", "", false
 }
 
-func CurrentCertificateNearingExpiry(c clock.Clock) Func {
+// CurrentCertificateNearingExpiry returns a policy function that can be used to check whether
+// an x509 cert currently issued for a Certificate should be renewed
+func CurrentCertificateNearingExpiry(c clock.Clock, defaultRenewBeforeExpiryDuration time.Duration) Func {
+
 	return func(input Input) (string, string, bool) {
-		if input.Certificate.Status.RenewalTime == nil {
-			return "", "", false
+
+		// Determine if certificate is nearing expiry solely by looking at the actual cert if it exists
+		// We assume that at this point we have called policy functions that check that
+		// input.Secret and input.Secret.Data exists (SecretDoesNotExist and SecretHasData)
```

Comment:
sorry for this nit! (dots help me parse comments faster 😇)

```suggestion
		// Determine if the certificate is nearing expiry solely by looking at the actual cert if it
		// exists. We assume that at this point we have called policy functions that check that
		// input.Secret and input.Secret.Data exists (SecretDoesNotExist and SecretHasData).
```